### ISSUE TITLE
Enhance the Example.java code to use a visible web browser

### ISSUE THREAD

**Issue by GoogleCodeExporter** [julianharty/web-accessibility-testing#5]
```
Sometimes we need to assess WCAG guidelines for pages protected by a login. One 
easy way to perform the assessment is to set a breakpoint when the URL is 
loaded to allow someone to manually login to the web site. If necessary the 
human can then navigate to the page to assess.

Then continue running the code which will obtain the current HTML of the target 
page and run the WCAG query.

Currently the example code processes the URL directly using Java's URL lib. 
This means a person cannot interact with the web site. So if we can create a 
version of the Example page that uses a visible web browser we can then improve 
the reach of our testing. I propose we use WebDriver which supports a wide 
range of web browsers.


```

Original issue reported on code.google.com by `julianharty` on 24 Nov 2011 at 11:52
